.github/workflows/ci.yml

```diff
@@ -7,6 +7,10 @@ on:
   pull_request:
     branches: [main]
 
+env:
+  NODE_VERSION: 16.x
+  DKG_TESTS_PATH: github.com/alicenet/alicenet/layer1/executor/tasks/dkg/tests
```

Comment:
Can just move this to the integration test job if we remove the callout for `go-unit-tests`